prover/Cargo.toml

```diff
@@ -9,15 +9,15 @@ repository = "https://github.com/0xPolygonMiden/miden-vm"
 categories = ["cryptography", "emulators", "no-std"]
 keywords = ["miden", "prover", "stark", "zkp"]
 edition = "2021"
-rust-version = "1.65"
+rust-version = "1.67"
 
 [features]
-concurrent = ["prover/concurrent", "std"]
+concurrent = ["processor/concurrent", "std"]
 default = ["std"]
-std = ["air/std", "processor/std", "prover/std", "log/std"]
+std = ["air/std", "processor/std", "log/std"]
```

Comment:
Do we not need to mention `winter-prover` here too?

Reply:
Hmm true, should have been there as well. Fixed!